# old map ID
string map_id

# new name
string name

---
